tests/de.rs

```diff
@@ -0,0 +1,24 @@
+#[macro_use]
+extern crate serde_derive;
+extern crate serde;
+extern crate serde_ini;
+
+use serde::Deserialize;
+use serde_ini::de::Deserializer;
+
+#[test]
+fn test_de_basic_ok() {
+    #[derive(Deserialize)]
+    struct Model {
+        key1: String,
+        key2: u32,
+    };
+    let input = "
+; Ignored comment
+key1=value1
+key2=255
+";
+    let bufrd = std::io::BufReader::new(input.as_bytes());
+    let parsed_str = Model::deserialize(&mut Deserializer::from_str(&input)).unwrap();
+    let parsed_bufrd = Model::deserialize(&mut Deserializer::from_bufread(bufrd)).unwrap();
+}
```

Comment:
Nice! An `assert_eq!()` for these against the expected `Model { key1: "value1", key2: 255 }` would fill this test out pretty well.